specification/dimensions/usageunit.md

```diff
@@ -0,0 +1,91 @@
+# Usage Unit
+
+Usage Unit is a measure of the amount of a resource or service was used or purchased. The Usage Unit for a particular line item may differ from the published Pricing Unit when providers use different units, unit increments, or columns for calculating cost. The UsageUnit column should not be used to calculate metrics related to any cost metric. This unit is commonly used when auditing or reconciling usage data.
```

Comment:
@ijurica - I added the sentence "The UsageUnit column should not be used to calculate metrics related to any cost metric."

Reply:
Similar cross-references will be included in PricingUnit and QuantityInPricingUnit specs. 

Couple of comments/suggestions available in one of the conversations in [PricingUnit PR](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/179) and VS1 team scheduled a 30 min discussion on this topic for Friday, at14:30 - 15:00 UTC. I'm well aware that this is a late notice. It would be great if you could join us.